src/components/navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import SearchIcon from "./svgs/search";
+import ProfileIcon from "./svgs/profile";
+import AppointmentsIcon from "./svgs/appointments";
+import ReceiptsIcon from "./svgs/receipts";
+import React from "react";
+
+type IconButtonProps = {
+    icon: React.ReactNode; 
+    label: string;
+};
+
+function IconButton({ icon, label }: IconButtonProps) {
+    return (
+        <div className="group text-white flex flex-col items-center p-4 rounded-mini hover:bg-teal-light">
```

Comment:
since they're buttons, use the button tag, helps with accessibility when using screen readers